nemo/collections/nlp/models/language_modeling/megatron/gpt_model.py

```diff
@@ -24,6 +24,7 @@
     parallel_lm_logits,
     scaled_init_method_normal,
 )
+from nemo.collections.nlp.parts import utils_funcs as utils
```

Comment:
can we do this import without aliasing?

Reply:
Sure but this is nlp utils module so it seems better to alias it. For example, if it was utils from other folders sure id use the full name but for the domains main utility module it seems excessive. This module should originally have been named just utils, i didn't change it cause its entrenched in multiple files already.

Reply:
Done